### PR TITLE
Do not print "Nothing found!" in bug bounty mode

### PR DESCRIPTION
**Pull request by Alireza-Sampour**
Related to issue #155

### CODE REVIEW

**Comment by bee-san** [comment]
Hey 👋 Why'd you close this? :) This looks like it could work and it'd count towards Hacktoberfest!

**Comment by Alireza-Sampour** [comment]
Hi, this breaks some tests, and unfortunately, I couldn't figure out how to change the code to pass tests.

File: pywhat/printer.py [hunk 1]
```diff
@@ -157,7 +157,7 @@ def print_raw(self, text: dict, text_input) -> str:
                         output_str += "\n[bold #D7Afff]Exploit: [/bold #D7Afff]" + i["Regex Pattern"]["Exploit"]
                     output_str += "\n\n"
 
-        if output_str == "":
+        if output_str == "" and self.bug_bounty_mode:
```
Comment (bee-san) [inline pywhat/printer.py:160]: I think you meant to say `not self.bug_bounty_mode` here. Try that? The tests should work :)

**Comment by Alireza-Sampour** [comment]
Thanks for your following up, I guess I should create another pull request (because when I couldn't figure out what's wrong with my changes I deleted the forked repository :( )

**Pull request closed without merging**